inst/help/raterAgreement.md

```diff
@@ -1,28 +1,32 @@
-Cohen's and Fleiss' kappa
+Rater Agreement
 ==========================
 
-Cohen's and Fleiss' kappa are coefficients that measure the agreement between raters on a nominal or ordinal scale (Cohen, 1960; Cohen, 1968; Fleiss, 1971). Cohen's kappa is limited to measure the agreement between two raters. Fleiss' kappa measures the agreement between two or more raters.
+Cohen's and Fleiss' kappa and Krippendorff's alpha are coefficients that measure the agreement between raters on a nominal or ordinal scale (Cohen, 1960; Cohen, 1968; Krippendorff, 1970; Fleiss, 1971).
+ Cohen's kappa is limited to measure the agreement between two raters. Fleiss' kappa and Krippendorff's alpha measure the agreement between two or more raters.
 
 *Rater* can refer to different judges, tests or other forms of rating here and in the section below.
 
 ### Input
 -------
 
-- Variables: The variables / columns to compute kappa for. Each variable corresponds to one rater, with different rows corresponding to different subjects being rated. 
-The maximum number of raters for Cohen's kappa is 2. If more than 2 variables/raters are entered, all possible pairs of the variables/raters will be calculated.
+- Variables: The variables / columns to compute kappa for. Each variable corresponds to one rater, with different rows corresponding to different subjects being rated. Note that for Krippendorff's alpha this is different from
+the common format, which has one rater per row and one column of measurements per subject. For Cohen's kappa the maximum number of raters  is 2. If more than 2 variables/raters are entered, all possible pairs of the variables/raters will be calculated. 
```

Comment:
Maybe rewrite this a little: "...Note that this data format is different from the common data format for Krippendorff's alpha, which has raters in rows and units of measurements (subjects) in columns...."